Release 3.8.6 is missing dependencies

Download the latest release from https://github.com/typetools/annotation-tools/releases/tag/3.8.6

Unzip the file.

Running `annotation-tools/annotation-file-utilities/scripts/extract-annotations` and other scripts results in:

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/plumelib/options/Options
	at scenelib.annotations.io.classfile.ClassFileReader.main(ClassFileReader.java:64)
Caused by: java.lang.ClassNotFoundException: org.plumelib.options.Options
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 1 more
```

--- a/annotation-file-utilities/build.xml
+++ b/annotation-file-utilities/build.xml
@@ -308,6 +308,7 @@
         <include name="annotation-file-format.html"/>
         <include name="annotation-file-format.pdf"/>
         <include name="annotation-file-utilities.jar"/>
+        <include name="annotation-file-utilities-all.jar"/>
         <include name="bin/**"/>
       </fileset>
     </copy>

